Add "Telemetry E2E" package

ctron opened the pull request:
This is a draft PR for the new "Telemetry E2" package. As outlined in #274

The current state is:

* [ ] Drogue IoT Device – Pending
* [ ] Eclipse ioFog – Issues with Kubernetes deployment: waiting for next 3.x pre-release
* [ ] Eclipse Kura – Performance issue in armv7 container: investigating Java 11 from AdoptOpenJDK
* [x] Drogue IoT Cloud
* [x] Eclipse Ditto
* [x] Eclipse Streamsheets

Additional tasks:

* [ ] Documentation

Changes from original plan:

* Drop the idea of using Raspberry Pi Zero: Support for armv7 is already challenging, the Zero however is an armv6 platform, which is even less supported in the Java world. Thus is may be too much effort this get this working properly, and switching to a Raspberry Pi 3+ model seems a reasonable compromise.

My proposal is to continue working on the documentation of the cloud-side parts, add a Kura emulator to the Kubernetes deployment for now and do a first "preview" (draft) release once this is accomplished.

ctron commented:
@calohmn I think it would be great if you (as project lead) could give a quick thumbs up (or comment otherwise) on the overall idea of adding this chart.

thjaeckle commented on packages/telemetry-e2e/values.yaml:
```diff
@@ -0,0 +1,182 @@
+
+ditto:
+  fullnameOverride: ditto
+  dittoTag: 2.1.0-M3
```
this can be dropped now in favor of using the latest 2.1 Ditto chart with final Ditto 2.1.0 in it

ctron commented:
I squashed the commits and extracted the homepage parts to another PR.

Unless there are any concerns, I would release a first alpha version of this chart in the next days.
